containers/podman-desktop
# feat: replace FontAwesome icon by a seal rocket loader

### What does this PR do?
replace FontAwesome icon by a seal rocket loader for the initialization
credits go to @mairin 

### Screenshot/screencast of this PR

https://github.com/containers/podman-desktop/assets/436777/324b5870-f6a0-4bc1-ac91-e8b45c374c90

### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/2732

### How to test this PR?

If you're not able to see for a long time the loader, you can toggle line 47
https://github.com/containers/podman-desktop/blob/3ed4717e5804c46d322c2e4f18b53fc7458a4946/packages/renderer/src/Loader.svelte#L47

by `if !false`


Change-Id: Id806b33548575b6924d4c8b6475b941721077e2a
